packages/connectivity/connectivity_platform_interface/README.md

```diff
@@ -0,0 +1,26 @@
+# connectivity_platform_interface
+
+A common platform interface for the [`connectivity`][1] plugin.
```

Comment:
nit: I don't think this is valid Github flavored markdown, I'm not seeing it render: https://github.com/flutter/plugins/blob/406bb57c724f3974c8c31a7f980071e5a702ff4c/packages/connectivity/connectivity_platform_interface/README.md. I'd just put an inline link to `pub.dev` here.

Reply:
I copied it from https://raw.githubusercontent.com/flutter/plugins/master/packages/url_launcher/url_launcher_platform_interface/README.md. Your link renders fine for me. What are you seeing?

Reply:
If you actually click on any of the links at https://github.com/flutter/plugins/blob/master/packages/connectivity/connectivity_platform_interface/README.md they 404. Can you open up a PR and fix this in a follow up?

Reply:
Ahh yeah, `connectivity/connectivity` will be created in the next PR. Will fix now. Thanks!